### PR TITLE
[MIG][website_sale_checkout_country_vat] Migrate to 10.0 

### PR DESCRIPTION
**Pull request by yajo**
@Tecnativa

### CODE REVIEW

File: website_sale_checkout_country_vat/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Configuration
+=============
+
+The default country that will appear is the one of the current user, or the one
+of the Public User assigned to the website, or the one of the company that owns
+the website. Change it in the corresponding place to get a sane default.
+
+If you want to have a sane default also in the country address, check the
+`website_sale_default_country
+<https://github.com/OCA/e-commerce/tree/8.0/website_sale_default_country>`_
```
Comment (chienandalu) [inline website_sale_checkout_country_vat/README.rst]: s/8.0/10.0

File: website_sale_checkout_country_vat/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# Copyright 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Website Sale Checkout Country VAT",
+    "summary": "Autocomplete VAT in checkout process",
+    "version": "10.0.1.0.0",
+    "category": "Website",
+    'website': 'http://www.tecnativa.com',
```
Comment (chienandalu) [inline website_sale_checkout_country_vat/__manifest__.py]: https

**Review by chienandalu — COMMENTED** [review]
Little comments. Runbot and Travis aren't catching this module's dependency. Rebase maybe?

**Comment by chienandalu** [comment]
Runbot and Travis are failing because: `Could not execute command 'sass'`.

**Comment by pedrobaeza** [comment]
@Yajo, for Travis, you need to define https://github.com/OCA/maintainer-quality-tools/blob/f54d6e33c5c48e6bd8051efe018e88ff3db669f8/sample_files/.travis.yml#L36

**Comment by lasley** [comment]
@pedrobaeza - the value in the sample file is actually the default, so that isn't the issue. That key would be used to set it back versions, or use the OS one.

The issue here is that we need [bootstrap-sass](https://github.com/OCA/website/blob/10.0/.travis.yml#L31)

**Comment by pedrobaeza** [comment]
But we have also an error in PhantomJS tests, so I figured out that the problem was that...

**Comment by lasley** [comment]
I believe that's because the page assets [weren't able to be bundled](https://travis-ci.org/OCA/e-commerce/jobs/247416866#L901), which caused the subsequent [Phantom error due to lack of assets defined in the bundle](https://travis-ci.org/OCA/e-commerce/jobs/247416866#L929)

**Comment by pedrobaeza** [comment]
OK, then let's go to get the dependency and see

**Comment by pedrobaeza** [comment]
Same issue as https://github.com/OCA/maintainer-quality-tools/issues/471

**Comment by lasley** [comment]
@pedrobaeza - this is fixed in v10. Just needs to rebase & remove the manual sass installation.

**Comment by lasley** [comment]
At least I hope that's the case...

**Comment by pedrobaeza** [comment]
@lasley I'm afraid not. There are still errors with PhantomJS and for sure in runbot...

**Comment by lasley** [comment]
Damn so I'm not sure if I know how to fix this then - the are no more install errors on the RVM/compass part. How do I replicate the error on Runbot?

**Comment by pedrobaeza** [comment]
@Yajo, any clue about the error?

```
AssertionError: None is not true : PhantomJS test completed without reporting success; the log may contain errors or hints.
```

**Comment by lasley** [comment]
This looks like the real error I think:
```
phantomjs: u"Could not get content for /website_sale_checkout_country_vat/static/src/js/change_country.js defined in bundle 'web.assets_frontend'.\n at http://127.0.0.1:8069/web/content/506-68a014f/web.assets_frontend.js:42 (in global code)\n(leaf frame on top)"
```

**Review by pedrobaeza — APPROVED** [review]
Now we get it!

The error is a bit cumbersome to be read, but @lasley hit it.

@chienandalu please review

**Review by chienandalu — APPROVED** [review]
Tested on rubot :+1:

**Pull request merged**